hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/validate/HazelcastSqlOperatorTable.java

```diff
@@ -263,6 +272,71 @@
 
     //#endregion
 
+    //#region Math functions.
+
+    public static final SqlFunction ABS = new SqlFunction(
+        "ABS",
+        SqlKind.OTHER_FUNCTION,
+        HazelcastReturnTypes.UNARY_MINUS,
+        new ReplaceUnknownOperandTypeInference(DECIMAL),
+        notAny(OperandTypes.NUMERIC_OR_INTERVAL),
+        SqlFunctionCategory.NUMERIC
+    );
+
+    public static final SqlFunction SIGN = new SqlFunction(
+        "SIGN",
+        SqlKind.OTHER_FUNCTION,
+        ReturnTypes.ARG0,
+        new ReplaceUnknownOperandTypeInference(DECIMAL),
+        notAny(OperandTypes.NUMERIC),
+        SqlFunctionCategory.NUMERIC
+    );
+
+    public static final SqlFunction RAND = new SqlFunction(
+        "RAND",
+        SqlKind.OTHER_FUNCTION,
+        ReturnTypes.DOUBLE,
+        new ReplaceUnknownOperandTypeInference(BIGINT),
```

Comment:
Shouldn't it be done the same way as for `DoubleFunction`?

Reply:
Changed to explicit inference